gateway-service/src/main/java/org/zowe/apiml/gateway/security/service/AuthenticationServiceUtils.java

```diff
@@ -10,16 +10,20 @@
 package org.zowe.apiml.gateway.security.service;
 
 import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.security.core.AuthenticationException;
+import org.zowe.apiml.security.common.token.QueryResponse;
 import org.zowe.apiml.security.common.token.TokenAuthentication;
 
 import javax.servlet.http.HttpServletRequest;
 import java.util.Optional;
 
 /**
- * Provide syntax sugar for HTTP based authentication
+ * Provide syntax sugar for {@link AuthenticationService}
  */
 @RequiredArgsConstructor
-public class HttpAuthenticationService {
```

Comment:
Also uncertain about this renaming. If anything I would look for other meaningful name, but Utils doesn't have much of a meaning helping to guide where to put functionality so that it is cohesive.

Reply:
Renamed as agreed